system_setup/client/controllers/wslconfbase.py

```diff
@@ -12,7 +12,11 @@ class WSLConfigurationBaseController(SubiquityTuiController):
 
     async def make_ui(self):
         data = await self.endpoint.GET()
-        return WSLConfigurationBaseView(self, data)
+        variant = "wsl_setup"
+        onsite_variant = getattr(self.app, "variant")
+        if onsite_variant is not None:
+            variant = onsite_variant
+        return WSLConfigurationBaseView(self, data, variant)
```

Comment:
Same, variant should not be needed in the Base View, it’s the same in both setup and reconfiguration

Reply:
This one is for In wsl_configuration mode the UI shows ”base”/”advanced” screens.

Reply:
See my other comment below, which should make this uneeded :)

Reply:
addressed